commit fac95526db1e39325917e0bacf913630660bbc46
Author: HaseenaSainul <h.sainul@metrological.com>
Date:   Tue Jan 21 15:34:10 2020 +0530

    FindNXCLIENT: find path of nxserver include of nxclient as well

diff --git a/cmake/modules/FindNXCLIENT.cmake b/cmake/modules/FindNXCLIENT.cmake
index 3315673..5f8730b 100644
--- a/cmake/modules/FindNXCLIENT.cmake
+++ b/cmake/modules/FindNXCLIENT.cmake
@@ -26,9 +26,12 @@
 # OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-find_path(LIBNXCLIENT_INCLUDE nexus_config.h
+find_path(NXCLIENT_INCLUDE nexus_config.h
         PATH_SUFFIXES refsw)
 
+find_path(NXCLIENT_NXSERVER_INCLUDE nxserverlib.h
+        PATH_SUFFIXES nxclient)
+
 find_library(LIBNXCLIENT_LIBRARY nxclient)
 
 if(EXISTS "${LIBNXCLIENT_LIBRARY}")
@@ -36,6 +39,7 @@ if(EXISTS "${LIBNXCLIENT_LIBRARY}")
 
     set(NXCLIENT_FOUND TRUE)
 
+    set(LIBNXCLIENT_INCLUDE ${NXCLIENT_INCLUDE} ${NXCLIENT_NXSERVER_INCLUDE})
     find_package_handle_standard_args(LIBNXCLIENT DEFAULT_MSG NXCLIENT_FOUND LIBNXCLIENT_INCLUDE LIBNXCLIENT_LIBRARY)
     mark_as_advanced(LIBNXCLIENT_LIBRARY)
 
